macos/FluentUI/Core/FluentUIResources.swift

```diff
@@ -12,10 +15,23 @@ public class FluentUIResources: NSObject {
 
 	/// The resource bundle contained within the FluentUI framework
 	@objc public static let resourceBundle: Bundle = {
+		#if SWIFT_PACKAGE
+		return Bundle.module
+		#else
 		guard let url = bundle.resourceURL?.appendingPathComponent("FluentUIResources-macos.bundle", isDirectory: true), let bundle = Bundle(url: url) else {
```

Comment:
even after the migraton, the cocoapod path can still use fluentuiresource-macos.bundle as is?

Reply:
Good catch, I need to update the podspec to include the new resources path.